packages/database/src/core/SyncPoint.ts

```diff
@@ -128,10 +130,37 @@ export class SyncPoint {
         new CacheNode(eventCache, eventCacheComplete, false),
         new CacheNode(serverCache, serverCacheComplete, false)
       );
-      view = new View(query, viewCache);
-      this.views.set(queryId, view);
+      return new View(query, viewCache);
     }
+    return view;
+  }
 
+  /**
+   * Add an event callback for the specified query.
+   *
+   * @param {!Query} query
+   * @param {!EventRegistration} eventRegistration
+   * @param {!WriteTreeRef} writesCache
+   * @param {?Node} serverCache Complete server cache, if we have it.
+   * @param {boolean} serverCacheComplete
+   * @return {!Array.<!Event>} Events to raise.
```

Comment:
```suggestion
   * @param query
   * @param eventRegistration
   * @param writesCache
   * @param serverCache Complete server cache, if we have it.
   * @param serverCacheComplete
   * @return Events to raise.
```

I just removed the Closure types in the rest of the SDK.